### DISCUSSION TITLE
터틀심, airobo py 파일 찾기

### DISCUSSION THREAD

**Discussion by minsuLee-kr** [Road-Balance/gcamp_ros2_basic#44] [Q&A] [answered]
airobo 파일은 찾아본 결과 
~/ros2_ws/src/gcamp_ros2_basic/airobo_gazebo/launch/ gazebo_world.launch 에 있는거 찾았는데
터틀심도 일일이 파일을 뒤지면서 찾으려 하는데 apt install 로 다운 받은 거라서 py 파일이 어디 있는 지도 모르겠습니다.
 이렇게 찾는게 아닌거같고 터미널 명령어 쳐서 찾는건지 궁금합니다.

강의 자료도 찾아봤는데 결국 못찾았습니다.

**Comment by kimsooyoung** [comment, marked as answer]
터틀심은 따로 코드가 있는 것이 아닙니다. apt install 로 다운받았기 때문에 코드 없이 실행 프로그램만 다운받아진 상황입니다.
